Signal/src/util/OWSScreenLockUI.m

```diff
@@ -32,6 +32,39 @@ typedef NS_ENUM(NSUInteger, ScreenLockUIState) {
 
 const UIWindowLevel UIWindowLevel_Background = -1.f;
 
+// OWSScreenBlockingWindow tries to become first responder
+// when presented to ensure that the input accessory is updated.
+@interface OWSScreenBlockingViewController : UIViewController
```

Comment:
I'm not sure whether we should merge this PR.  It may not be necessary, and it doesn't fix a known repro.  

However, we've recently learned (in the case of call view) that a reliable way to update the input accessory is to switch first responders - not just `resignFirstResponder` but to ensure that there is a new first responder.

So I've opened this PR in the spirit of being defensive.